### PR TITLE
Adding new structural test of remeshing

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Review by annasophie91 — APPROVED** [review]

File: applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h [hunk 1]
```diff
@@ -154,7 +154,7 @@ class NodalValuesInterpolationProcess
             Vector ShapeFunctions;
             Element::Pointer pElement;
             
-            const bool IsFound = PointLocator.FindPointOnMeshSimplified(itNode->Coordinates(), ShapeFunctions, pElement, mMaxNumberOfResults);
+            const bool IsFound = PointLocator.FindPointOnMeshSimplified(itNode->Coordinates(), ShapeFunctions, pElement, mMaxNumberOfResults, 5.0e-2);
```
Comment (pooyan-dadvand) [inline applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: This is a very large tolerance! Is it correct?

Comment (loumalouomega) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: Indeed, but didn't work otherwise

Comment (pooyan-dadvand) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: It look like you have some "algorithmic" problem hidden behind....

Comment (loumalouomega) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: Could be, I just know that the problem is solved modifying the tolerance. This problem already happened in the past

**Comment by RiccardoRossi** [comment]
+1 for @pooyandadvand comment

File: applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h (hunk 1 above)
Comment (RiccardoRossi) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: Can u see if by using a small tolerance it works? If it does not we shall see why it doesnt

Comment (loumalouomega) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: I tried with all the intermediate tolerances, even 1.0e-3 didn't work

Comment (RiccardoRossi) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: It would be work to see what is failing...

Comment (pooyan-dadvand) [reply applications/MeshingApplication/custom_processes/nodal_values_interpolation_process.h:157]: :+1:

**Pull request merged**